sequence_data_processing.py

```diff
@@ -133,7 +133,7 @@ def __init__(self, input_configuration, debug=False, output="output", j=1, gener
             # Read configuration from the argument dict
             self.input_configuration_file = None
             self._campaign_configuration = input_configuration
-            general_args = {'output': output, 'seed': seed, 'j': j, 'debug': debug,
+            general_args = {'output': output, 'j': j, 'debug': debug,
```

Comment:
Just confirming: this was removed because the `seed` variable is not defined here, correct?

Reply:
Definitely: it was causing the program to crash since it was not updated as the from-string configuration version